mne/io/brainvision/brainvision.py

```diff
@@ -383,56 +397,92 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
     # set to zero.
     settings = settings.splitlines()
     idx = None
+
     if 'Channels' in settings:
         idx = settings.index('Channels')
         settings = settings[idx + 1:]
+        hp_col, lp_col = 4, 5
         for idx, setting in enumerate(settings):
             if re.match('#\s+Name', setting):
                 break
             else:
                 idx = None
 
+    # If software filters are active, then they override the hardware setup
+    # But we still want to be able to double check the channel names
+    # for alignment purposes, we keep track of the hardware setting idx
+    idx_amp = idx
+
+    if 'S o f t w a r e  F i l t e r s' in settings:
+        idx = settings.index('S o f t w a r e  F i l t e r s')
+        for idx, setting in enumerate(settings[idx + 1:], idx + 1):
+            if re.match('#\s+Low Cutoff', setting):
+                hp_col, lp_col = 1, 2
+                warn('Online software filter detected. Using software '
+                     'filter settings and ignoring hardware values')
```

Comment:
Does the BrainVision software filtering actually change what's written to disk? I thought that it only changed what was displayed during acquisition. But it's been a while since I tinkered with the BV interface...